src/software/SfM/main_ComputeFeatures_OpenCV.cpp

```diff
@@ -9,6 +9,7 @@
 // The <cereal/archives> headers are special and must be included first.
 #include <cereal/archives/json.hpp>
 
+#include "openMVG/features/regions_factory_io.hpp"
 #include "openMVG/image/image_io.hpp"
```

Comment:
Please order the include in alphabetical order.

Reply:
@pmoulon Resolved. I did not realise you are sorting them by the name of a file. I'm usually sorting them by whole path.